datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -693,32 +694,6 @@ impl LogicalExtensionCodec for TopKExtensionCodec {
     }
 }
 
-#[test]
-fn scalar_values_error_serialization() {
-    let should_fail_on_seralize: Vec<ScalarValue> = vec![
-        // Should fail due to empty values
-        ScalarValue::Struct(
-            Some(vec![]),
```

Comment:
Not supposed to create empty array, so just removed the test

Reply:
Move to `round_trip_scalar_values`, since it is able to serialized

```
        ScalarValue::try_from(&DataType::Struct(Fields::from(vec![
            Field::new("a", DataType::Int32, true),
            Field::new("a", DataType::Boolean, false),
        ])))
        .unwrap(),
```